{{cookiecutter.project_slug}}/config/settings/base.py

```diff
@@ -226,7 +226,7 @@
 EMAIL_BACKEND = env(
     "DJANGO_EMAIL_BACKEND", default="django.core.mail.backends.smtp.EmailBackend"
 )
-# https://docs.djangoproject.com/en/2.2/ref/settings/#email-timeout
+# https://docs.djangoproject.com/en/dev/ref/settings/#email-timeout
```

Comment:
3.0?

Reply:
All our links are to the dev version so this is to make this link consistent with the rest. We should probably link to the stable one, but I'd say it's a separate issue. What do you think?